src/components/TagInput/TagInput.tsx

```diff
@@ -42,19 +44,21 @@ export const TagInput = forwardRef<HTMLInputElement, TagInputProps>(({
   return (
     <div>
       <TextInput
-        type="text"
-        value={input}
-        label={label}
         hint={hint}
+        label={label}
         onChange={(event: FormEvent<HTMLInputElement>) => setInput((event?.target as HTMLTextAreaElement)?.value)}
         onKeyDown={(event: KeyboardEvent<HTMLInputElement>) => handleKeyDown(event)}
+        placeholder={placeholder}
+        type="text"
+        value={input}
       />
       <TagGroup>
         {(values ?? []).map((tag) => (
           <Tag
-            key={tag}
             className="fr-mr-1w"
+            key={tag}
             onClick={() => handleDeleteClick(tag)}
+            title={`Tag ${tag}`}
```

Comment:
I think this is useless for accessibility but not sure, havn't seen that in other tagInput components

Reply:
It seems to be necessary for accessibility but the text should be something like "Word (tag) selected, press Enter to delete it"